### DISCUSSION TITLE
How do I invite a colleague or friend to join the Beta?

### DISCUSSION THREAD

**Discussion by bbonnotte-stoic** [stoic-doc/Community#26] [Documentation]
*`Provisioning`* `Beta`

**Comment by ghalimi** [comment]
To invite a colleague or friend to join the Beta, simply submit an `Issue` from the Support Panel in the user interface, with your colleague's or friend's email address and LinkedIn Profile. Please do not make this submission from this repository's Issues, as we do not want to make this personal information publicly available. Thank you so much for helping us spread the word!

**Reply by bbonnotte-stoic** [reply]
Thanks @ghalimi for the answer !
